docs/src/SearchField.doc.js

```diff
@@ -48,6 +48,10 @@ card(
         syntheticEvent: SyntheticEvent<HTMLInputElement>
       }) => void`,
       },
+      {
+        name: 'onKeyDown',
+        type: '({ event: SyntheticKeyboardEvent<HTMLInputElement>, value: string }) => void',
```

Comment:
Could you add a description fo the prop here?

Reply:
Done